src/meta/src/hummock/vacuum.rs

```diff
@@ -103,7 +103,7 @@ where
 
         // Dispatch the vacuum task
         let mut batch_idx = 0;
-        let batch_size = 32usize;
+        let batch_size = 500usize;
```

Comment:
AWS restricts the number of objects per request to 1000, refer to `delete_objects` function of s3.rs, are they the same?
```rust
const MAX_LEN: usize = 1000;
```

Reply:
Yes. And the delete_objects has already split large input into multiple s3 calls. So it's OK to set arbitrary batch_size here.